IR_Turret/IR_Turret.ino

```diff
@@ -334,52 +346,45 @@ void stepFire(){
   }
 }
 
-void defineMode(){
-    if (IRcompare(numberpulses, VolDownSignal,sizeof(VolDownSignal)/4)) {
+void defineMode(unsigned long value) {
+  switch (value) {
+    case 68: // Example IR code for VolDownSignal
       Serial.println("Rotate Left!");
       Serial.println(" ");
       mode = "LEFT_TURN";
-    }
-
-    if (IRcompare(numberpulses, VolUpSignal,sizeof(VolUpSignal)/4)) {
+      break;
+    case 64: // Example IR code for VolUpSignal
       Serial.println("Rotate Right!");
       Serial.println(" ");
       mode = "RIGHT_TURN";
-    }
-
-    if (IRcompare(numberpulses, PrevSignal,sizeof(PrevSignal)/4)) {
+      break;
+    case 7: // Example IR code for PrevSignal
       Serial.println("Previous");
       mode = "TILT_DOWN";
-    }
-
-    if (IRcompare(numberpulses, NextSignal,sizeof(NextSignal)/4)) {
+      break;
+    case 21: // Example IR code for NextSignal
       Serial.println("Next");
       mode = "TILT_UP";
-    }
-
-    if (IRcompare(numberpulses, PlaySignal,sizeof(PlaySignal)/4)) {
+      break;
+    case 67: // Example IR code for PlaySignal
       Serial.println("Fire All!");
       mode = "FIRE_ALL";
-    }
-
-    if (IRcompare(numberpulses, ChannelSignal,sizeof(ChannelSignal)/4)) {
-      Serial.println("Channel");
+      break;
+    case 70: // Example IR code for ChannelSignal
+      Serial.println("Fire!");
       mode = "FIRE_SINGLE";
-    }
-    if (IRcompare(numberpulses, NumberOne,sizeof(NumberOne)/4)) {
-      Serial.println("Number 1");
+      break;
+    case 12: // Example IR code for NumberOne
+      Serial.println("Step Fire!");
       mode = "STEP_FIRE";
-    }
-    // if (IRcompare(numberpulses, NumberTwo,sizeof(NumberTwo)/4)) {
-    //   Serial.println("Number 2");
-    //   mode = "CIRCULAR_FIRE";
-    // }
-    // if (IRcompare(numberpulses, NumberThree,sizeof(NumberThree)/4)) {
-    //   Serial.println("Number 3");
-    //   mode = "SWEEP_TURRET";
-    // }
-    if (IRcompare(numberpulses, NumberFour,sizeof(NumberFour)/4)) {
-      Serial.println("Number 4");
+      break;
+    case 8: // Example IR code for NumberFour
+      Serial.println("Shake!");
       mode = "HEAD_SHAKE";
-    }
+      break;
+    default:
+      Serial.println(value);
```

Comment:
This line allows all uncaught IR codes to pass through to the serial console, where they can easily be copy/pasted back into the code.